documentation/docs/user-guide/user-officer/user-officer.md

```diff
@@ -0,0 +1,114 @@
+# User Officer
+
+User officer
+
+Welcome to the User Office documentation for **User Officers!**
+
+
+![workflow](images/call.png){ width="300" } ![workflow](images/workflow.png){ width="300" }
```

Comment:
The path should be `/assets/images/...`

Reply:
Thanks for the helpful suggestions Janos!